### ISSUE TITLE
Add subsubsection in docs. 

### ISSUE THREAD

**Issue by rocky** [Mathics3/mathics-core#1030]
Mathics3 modules in documentation are shifted down a level. In order to support "Guide" sections in Mathics3 modules, we would need to add a subsubsection support in LaTeX. 

Consider https://reference.wolfram.com/language/guide/LinguisticData.html

![linguistic-data](https://github.com/Mathics3/mathics-core/assets/8851/e51f90f2-1281-44c5-a238-c500ca9b083b)

We currently can't capture this structure in our documents because `WordTranslation`, `Alphabet`, `LanguageIdentify` etc. are at the subsubsection level. 

Because right now document structure follows module structure, we _also_ can't create a module in the Mathics3 Module for "Languages and Translation" and "Textual Analysis" and have its appear in the LaTeX doc. 

Two other things. 

Having subsubsections might be also useful outside of Mathics3 Modules.

Note that in Wolfram documentation Symbols are at multiple levels: `WordData` is at a different level than `WordTranslation`.  If you want to think of there being  and "anonymous" section or subsection", note that there is more than one of these anonymous (sub)sections

**Comment by mmatera** [comment]
@rocky, actually, we still have "Subsections". What I have changed is that "GuideSections" are Sections, so they contain "Subsections" instead of sections. 
Symbols in `pymathics.natlang.translation` should be accommodated in the current "mathics-code/master" as Subsection of a GuideSection "Translation" inside the chapter "Natural Language" of the "Mathics Modules"  Part.

I didn't check this package (I just had installed Pymathics.Graph when I run the tests) but it should work exactly the same. I will check what happens with natlang during the weekend.

**Comment by rocky** [comment]
From a code organization standpoint, it would be good to start supporting a format like that used in https://github.com/Mathics3/pymathics-natlang/pull/19/files .   What you want to _call_ the levels of the hierarchy  in the LaTeX doc is up for discussion. However in the LaTeX markup the nesting level is subsubsection.

**Comment by mmatera** [comment]
OK. My proposal for this would be to have three guides in the "Natural Language" chapter:
* Linguistic Data: Include (as a documentation entry, an explanation of the content, and a list)    
* Languages & Translations: contains the symbols in  lang_trans.py as subsections
* Translation: contains the symbols in lang_trans.py as subsections.

**Comment by rocky** [comment]
> OK. My proposal for this would be to have three guides in the "Natural Language" chapter:
> 
> * Linguistic Data: Include (as a documentation entry, an explanation of the content, and a list)
> * Languages & Translations: contains the symbols in  lang_trans.py as subsections
> * Translation: contains the symbols in lang_trans.py as subsections.

Yes, that makes sense. The problem of this issue/feature is that right now we can't extract docs for the lowest level.

**Comment by mmatera** [comment]
This is how Natlang looks (from its master branch)  when I remove the other parts and chapters from documentation.tex:

![image](https://github.com/Mathics3/mathics-core/assets/2862288/42174bfb-0c2d-481a-8a59-e042a9ff87e9)

During the weekend I will try to adjust this.
